doc/md_src_files/065_changelog.md

```diff
@@ -0,0 +1,60 @@
+
+## Version 2 Changelog
+
+This section summarizes the changes between SSVC version 2 and [SSVC version 1.1](https://weis2020.econinfosec.org/wp-content/uploads/sites/8/2020/06/weis20-final6.pdf) as published at the Workshop on the Ecnomics of Information Security (WEIS 2020).
+The details of what changes were made can be viewed on the SSVC GitHub [issues](https://github.com/CERTCC/SSVC/issues?q=is%3Aissue+is%3Aclosed+project%3ACERTCC%2FSSVC%2F1) closed under the SSVC v2 Development project.
+We addressed about 60 issues.
+About 10 issues identified "bugs" or errors in version 1.1.
+About 20 issues improved documentation of tools or improved clarity of document text.
+The remaining 30 issues were focused on enhancing SSVC based on feedback received on version 1, though several of the bug fixes and documentation improvements also provided improvements.
+This section focuses on changes that provided enhancements.
+
+### Coordination stakeholder
+
+Version 1 only considered two stakeholders: those who make software, and those who use information systems.
+Version 2 introduces a coordinator stakeholder and two distinct decisions for that stakeholder group: vulnerability intake triage and publication about a vulnerability.
+These decisions use some existing decision points, but also introduce six new decision points to support coordinators in making these decisions.
+The coordinator stakeholder is based on CERT/CC's experience coordinating vulnerabilities.
+
+### Improvements to decision points
+
+Version 1 had a decision point for well-being impact that was shared between **supplier** and **deployer** stakeholders.
+Since these types of stakeholder have access to quite different amounts of information about safety and well-being, Version 2 splits this concept into [*Public Safety Impact](#public-safety-impact) and [*Situated Safety Impact*](#situated-safety-impact).
```

Comment:
This sentence could use some rewording.  (It sounds like the amount of information is the focus.)
Since these types of stakeholder have access to quite different amounts of information about safety and well-being, Version 2 splits this concept into [*Public Safety Impact](#public-safety-impact) and [*Situated Safety Impact*](#situated-safety-impact).

Consider:
Since these types of stakeholder are focused on different of information regarding safety and well-being, Version 2 splits this concept into [*Public Safety Impact](#public-safety-impact) and [*Situated Safety Impact*](#situated-safety-impact).